cupy/core/internal.pxd

```diff
@@ -31,3 +31,9 @@ cpdef slice complete_slice(slice slc, Py_ssize_t dim)
 cpdef tuple complete_slice_list(list slice_list, Py_ssize_t ndim)
 
 cpdef size_t clp2(size_t x)
+
+ctypedef unsigned short _float16
+
+cpdef unsigned short to_float16(float f)
+
+cpdef float to_float(unsigned short v)
```

Comment:
This signature is misleading.  Is there a reason we cannot `cpdef float to_float(_float16 v)`?  How about `from_float16`?